cpp_macros/src/lib.rs

```diff
@@ -193,8 +223,8 @@ NOTE: They cannot be generated by macro expansion."#,
     let mut call_args = Vec::new();
     for (i, capture) in closure.sig.captures.iter().enumerate() {
         let written_name = &capture.name;
-        let mac_name: Ident = format!("$var_{}", written_name).into();
-        let mac_cty: Ident = format!("$cty_{}", written_name).into();
+        let mac_name = Ident::new(&format!("var_{}", written_name), Span::call_site());
```

Comment:
It'd be nice if we could save the real span of our input written_name identifiers.